### PR TITLE
update

### PR DESCRIPTION
**Pull request by saaiful**
ডাটাবেস ডিজাইন যুক্ত করা হয়েছে

### CODE REVIEW

**Comment by thesabbir** [comment]
@nuhil bhai what's your thought on cover photo?

**Comment by nuhil** [comment]
@saaiful Please do not make so many redundant commits with small changes. It will be fine if you make changes locally in the chapters and finally to summary file then create a pull request.

**Comment by nuhil** [comment]
@thesabbir What cover photo you are talking about? Would you like to add one?

**Comment by thesabbir** [comment]
@nuhil see file changed

**Comment by thesabbir** [comment]
@nuhil [link](https://github.com/saaiful/sql.howtocode.com.bd/blob/ba463a7dc0eabe9ba24321d4304d6bff12553234/cover.jpg)

**Comment by nuhil** [comment]
@saaiful You have been added as a collaborator. I also think, you should place a cover photo without your name at this stage. Our automated contribution API shows everyones contribution count. Though, you can put your bame statically once this course is completed and your contribution is more than 90% on it.

**Comment by nuhil** [comment]
@saaiful Rather you can put your name on the bottom of each chapter you are writing by yourself like me and masnun are doing on python course.

**Comment by saaiful** [comment]
Something went wrong with gitbook editor , it's eating bangla word while typing. I solved the cover issue. Also when i press ctrl+s it makes a commit, switching to atom's gitbook editor.

File: introduction.md [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
+১৯৭০ দশকের প্রথম দিকে [Donald D. Chamberlin](https://en.wikipedia.org/wiki/Donald_D._Chamberlin) এবং [Raymond F. Boyce](https://en.wikipedia.org/wiki/Raymond_F._Boyce) আইবিএমের "System R" নামক রিলেশনাল ডাটাবেজ এ ব্যাবহারের জন্য একটি লাংগুয়েজ তৈরি শুরু করেন। তারা লাংগুয়েজটির প্রথম ভার্সনটির নাম দেন SEQUEL \(Structured English Query Language\) যা পরবর্তিতে ট্রেডমার্ক জনিত সমস্যার কারনে পরিবর্তিত হয়ে SQL \(Structured Query Language\) হয়।
 
- এসকিউএল হচ্ছে Edgar F. Codd এর রিলেশনাল মডেল এর উপর ভিত্তি করে বানানো প্রথম ল্যাংগুয়েজ। যদিও এটি তখন তার রিলেশনাল মডেলের সাথে পুরোপুরি সামঞ্জস্য পূর্ন ছিল না। পরবর্তীতে এটিই বিশ্বের সবচেয়ে বহুল ব্যাবহৃত ডাটাবেজ লাংগুয়েজে পরিনত হয়।
+এসকিউএল হচ্ছে [Edgar F. Codd](https://en.wikipedia.org/wiki/Edgar_F._Codd) এর রিলেশনাল মডেল এর উপর ভিত্তি করে বানানো প্রথম ল্যাংগুয়েজ। যদিও এটি তখন তার রিলেশনাল মডেলের সাথে পুরোপুরি সামঞ্জস্য পূর্ন ছিল না। পরবর্তীতে এটিই বিশ্বের সবচেয়ে বহুল ব্যাবহৃত ডাটাবেজ লাংগুয়েজে পরিনত হয়।
 
-##এসকিউএল কেন শিখবেন ?
+## এসকিউএল কেন শিখবেন ?
```
Comment (thesabbir) [inline introduction.md:8]: can we remove point 1,2,3?
I think they are not educational rather career advice.

Comment (saaiful) [inline introduction.md:8]: Those are valid points, it is important to know why some one learn something like SQL.

**Comment by howtocode-dev** [comment]
As you both are collaborators on this repository. Please, resolve the git conflict and merge the pull request.

**Comment by thesabbir** [comment]
I'll merge it tonight :)

**Pull request merged**